src/idl_gen_python.cpp

```diff
@@ -38,6 +40,43 @@ static std::string TypeName(const FieldDef &field);
 // Hardcode spaces per indentation.
 const std::string Indent = "    ";
 
+const std::string keywords_[] = {
```

Comment:
This globally allocates a whole bunch of strings, even if they're not used. Make them `const char *`.. see `idlgencpp.cpp` for an example.